draft-dukhovni-tls-dnssec-chain.xml

```diff
@@ -320,7 +321,7 @@
 
       <t>
     The order of returned RRsets is unspecified and a TLS client MUST NOT
-    assume any ordering of RRsets.
+    assume any ordering of RRsets other than that RRsigs follow each RRset.
```

Comment:
I've now seen multiple cases of RRset not immediately followed by its RRSIG from nameservers, so we'll not make this change...